contributingGuides/STYLE.md

```diff
@@ -684,125 +574,9 @@ When writing a stateless component you must ALWAYS add a `displayName` property
 
 ## Stateless components vs Pure Components vs Class based components vs Render Props - When to use what?
 
-*_1. Stateless components: Used when you don't need to maintain state or use lifecycle methods._*
-
-In many cases, we create components that do not need to have a state, lifecycle hooks or any internal variables. In other words just a simple component that takes props and renders something presentational. But often times we write them as class based components which come with a lot of cruft (for e.g., it has a state, lifecycle hooks and it is a javascript class which means that React creates instances of it) that is unnecessary in many cases.
```

Comment:
This section is irrelevant because function components now can have "state" so they are not "stateless"